Makefile

```diff
@@ -44,6 +44,20 @@ up-local: .install-fqdn certificates/domain.crt certificates/domain.key .create-
 	bash scripts/local-deploy.sh
 	@$(MAKE) info-local
 
+.PHONY: up-devel
+up-devel: .install-fqdn certificates/domain.crt certificates/domain.key .create-secrets ## deploy osparc ops stacks and simcore
+	echo -n "Ensure osparc-simcore is running the very first time"

```

Comment:
MINOR: Avoid echo and use ``#`` to comment parts as we discussed
MINOR: if every step is commented , you can prefix most of the commands with ``@``

Reply:
removed line